.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ jobs:
       - checkout
       - add_ssh_keys:
           fingerprints:
-            - "08:05:8e:1f:ff:2e:4d:59:60:3e:a2:d5:e3:e2:b8:68"
+            - "2c:e8:55:f8:43:47:17:d6:b3:0d:15:23:72:27:13:ad"
             - "2f:37:66:17:11:ae:f9:0f:9e:52:98:41:fe:d0:0c:78"
       - run:
           command: bash .circleci/deploy-prod.sh
```

Comment:
This still references `deploy-prod.sh`, which does not exist.

Reply:
Changed to QA deploy